libs/core/src/lib/shellbar/user-actions-menu/components/user-actions-menu-header/user-actions-menu-header.component.ts

```diff
@@ -0,0 +1,94 @@
+import {
+    AfterContentInit,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    Host,
+    Input,
+    OnDestroy,
+    Optional,
+    ViewEncapsulation
+} from '@angular/core';
+import { Subscription } from 'rxjs';
+
+import { ShellbarUser } from '../../../model/shellbar-user';
+import { UserActionsMenuService } from '../../services/user-actions-menu.service';
+import { UserActionsMenuComponent } from '../user-actions-menu/user-actions-menu.component';
+import { UserActionsSubmenuComponent } from '../user-actions-submenu/user-actions-submenu.component';
+
+@Component({
+    selector: 'fd-user-actions-menu-header',
+    templateUrl: './user-actions-menu-header.component.html',
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class UserActionsMenuHeaderComponent implements AfterContentInit, OnDestroy {
+    /** Whether header component has removed borders */
+    @Input()
+    showMainHeader = true;
+
+    /** @hidden */
+    get title(): string {
+        const activeSubmenu = this._menuService?.getActiveSubmenu();
+
+        return activeSubmenu.title;
+    }
+
+    /** @hidden */
+    get user(): ShellbarUser {
+        return this._menuService?.menu.user;
+    }
+
+    /** @hidden */
+    _isSubmenu = false;
+
+    /** @hidden */
+    _compact = false;
+
+    /** @hidden */
+    private _menuService: UserActionsMenuService;
+
+    /** @hidden */
+    private subscription = new Subscription();
+
+    /** @hidden */
+    constructor(
+        /** @hidden */
+        private readonly _cd: ChangeDetectorRef,
+        /** @hidden */
+        @Optional() @Host() private readonly _manuComponent: UserActionsMenuComponent,
```

Comment:
```suggestion
        @Optional() @Host() private readonly _menuComponent: UserActionsMenuComponent,
```
same with `_submanuComponent` --> `_submenuComponent`